### PR TITLE
Cleans job.dm

### PR DESCRIPTION
**Pull request by PKPenguin321**
- ~~Removes unreachable code~~
- ~~Removes unneeded var declaration (var/new_mob)~~
- Removes a semicolon (?!?!?!?!)
- Renames some 1-letter vars to be readable
- Adds spacing where appropriate
- Rearranges `to_chat(M, "<b>You are the [rank].</b>")` to be before the `if(job)` check instead of being in the MIDDLE of the `if(job)` check which necessitated a SECOND `if(job)` check for literally no reason

You may notice this branch is named `job_sounds` but the PR is not job sounds. This code is why.

This PR is tested and works on a local server.

:cl:
code: Cleaned some very confusing code in subsystem/job.dm
/:cl:

### CODE REVIEW

File: code/controllers/subsystem/job.dm [hunk 1]
```diff
@@ -432,23 +433,18 @@ SUBSYSTEM_DEF(job)
...
 
+	to_chat(M, "<b>You are the [rank].</b>")
 	if(job)
-		var/new_mob = job.equip(H, null, null, joined_late , null, M.client)
-		if(ismob(new_mob))
-			H = new_mob
-			if(!joined_late)
-				N.new_character = H
-			else
-				M = H
```
Comment (LemonInTheDark) [inline code/controllers/subsystem/job.dm]: Silicons override equip() and return a new mob, which then replaces the old mob that was "robotized" and qdel'd.
[Shitcode](https://github.com/tgstation/tgstation/blob/master/code/modules/jobs/job_types/cyborg.dm#L17)
[Robotized](https://github.com/tgstation/tgstation/blob/master/code/modules/mob/transform_procs.dm#L389)
It really should be docced or refactored.

Comment (PKPenguin321) [reply code/controllers/subsystem/job.dm]: kill me

Comment (LemonInTheDark) [reply code/controllers/subsystem/job.dm]: not on your life

**Comment by spookydonut** [comment]
What parts of this are unreachable?  Asking for a friend https://github.com/SpaceManiac/SpacemanDMM/pull/123

**Comment by PKPenguin321** [comment]
Turns out none of it, will fix later

File: code/controllers/subsystem/job.dm (hunk 1 above)
Comment (81Denton) [reply code/controllers/subsystem/job.dm]: when this shitcode is in ashes, then you have my permission to die

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**